### DISCUSSION TITLE
Product roadmap

### DISCUSSION THREAD

**Discussion by mikaelmoller** [formium/formium#60] [General]
Hi - we are considering using formium in my company. But we can't really go all in on this before all form elements are available. Are you able to share your product roadmap?

**Comment by sparebytes** [comment]
Fields I really care about (Number, Date) have been "coming soon" for almost a year now. Any ETA on when we'll get these? Thanks.

**Reply by bureauberg** [reply]
Seems like Formium has been abandoned? Too bad, I like the concept.

**Comment by matthew-inamdar** [comment]
Same query here - although the Net Promoter Score is the field we're interested in.
